docs/reference/openapi-rules.md

```diff
@@ -1,3 +1,546 @@
-## OpenAPI Rules
+# OpenAPI Rules
 
-Coming soon...
\ No newline at end of file
+Spectral has three rulesets built in: 
+
+- [oas](#oas)
+- [oas2](#oas2)
+- [oas3](#oas3)
+
+_OAS is shorthand for OpenAPI Specification._
+
+In your ruleset file you can add `extends: "spectral:oas"` and you'll get everything from `spectral:oas2` and `spectral:oas3`, meaning you can lint either type of document and only the relevant rules will apply thanks to [formats](../getting-started/rulesets.md#formats).
```

Comment:
Hey isn't `spectral: oas` an "intersection" of the oas2 & oas3 sets? 

> you'll get everything from `spectral:oas2` and `spectral:oas3`
^ this suggests that it's a "union", no?

Reply:
Not sure what the question is here, so I'll just throw out some extra context. 

I've been trying to get spectral to support one oas ruleset and have oas2 and oas3 rules included, and it's finally getting done thanks to #445 and #479. These docs are talking like that PR is merged, because it'll need to be for Spectral v4.1 to happen. 

No longer will customers be forced to detect the version of the document _then_ apply the right ruleset. We should be doing that for them. It's been an issue ever since Spectral v3.

Reply:
Oh ok, I get it, I was thinking in terms of the older code which was something like:
- `oas` = common oas2 and oas3 rules
- `oas2` = `oas` + some oas2-specific rules
- `oas3` = `oas` + some oas3-specific rules

Reply:
That is correct historically, now and in the future. No change there.